examples/windows/dcos-hybrid.json.env

```diff
@@ -0,0 +1 @@
+export EXPECTED_NODE_COUNT=2
```

Comment:
Why do you need this?
By default we calculate the node count by parsing api model file.